draft-rfcs/resources/RFC-pre-rfc-process.md

```diff
@@ -0,0 +1,41 @@
+# Pre-RFC Process RFC
+
+# Summary
+[summary]: #summary
+
+This RFC introduces a pre-RFC process prior to submitting a formal RFC.
+
+# Motivation
+[motivation]: #motivation
+
+Navigating the current RFC process can be very intimidating, especially for newer community members. It is very discouraging for someone to spend a lot of time and effort on creating an RFC, then seeing it get rejected quickly for a reason that would have been clear if they had talked to someone on the relevant team first. There is little sense in writing a full, formal RFC without bringing a team member onboard first. The intent of this pre-RFC process is to help produce higher quality and more relevant RFCs that are more likely to be accepted. 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+This RFC introduces two roles:
+* **shepherd** - (add more here)
+* **liason** - this is a member of the team most relevant to the proposed change. This person should be called in as needed to work with the person composing the RFC, give feedback, and provide guidance throughout the pre-RFC process.
```

Comment:
Are both these roles team members from the relevant team? Why have two roles rather than one?

Reply:
Clarification is coming in the next draft :)